common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -45,21 +45,27 @@
 
   private static final boolean unaligned;
   static {
-    boolean _unaligned;
-    // use reflection to access unaligned field
-    try {
-      Class<?> bitsClass =
-        Class.forName("java.nio.Bits", false, ClassLoader.getSystemClassLoader());
-      Method unalignedMethod = bitsClass.getDeclaredMethod("unaligned");
-      unalignedMethod.setAccessible(true);
-      _unaligned = Boolean.TRUE.equals(unalignedMethod.invoke(null));
-    } catch (Throwable t) {
-      // We at least know x86 and x64 support unaligned access.
+      boolean _unaligned;
```

Comment:
I think its to do with my local setup, i dont see the issues in Intellij, sorry! :( 

ill revert my changes and try again